src/sgame/sg_cmds.cpp

```diff
@@ -3497,6 +3497,98 @@ void Cmd_Reload_f( gentity_t *ent )
 	}
 }
 
+/*
+=================
+Cmd_TeamStatus_f
+=================
+*/
+void Cmd_TeamStatus_f( gentity_t * ent )
+{
+	int builders = 0;
+	gentity_t *tmp;
+	struct buildable_status_t
+	{
+		int count;
+		int health;
+		bool spawned;
+	} structures[ BA_NUM_BUILDABLES ] = {};
+
+	if ( g_teamStatus.Get() <= 0 ) 
+	{
+		trap_SendServerCommand( ent - g_entities, "print \"teamstatus is disabled.\n\"" );
+		return;
+	}
+
+	if ( ent->client->pers.namelog->muted ) 
+	{
+		trap_SendServerCommand( ent - g_entities, "print \"You are muted and cannot use message commands.\n\"" );
+		return;
+	}
+
+	if ( level.team[ ent->client->pers.team ].lastTeamStatus
+		&& ( level.time - level.team[ ent->client->pers.team ].lastTeamStatus ) < g_teamStatus.Get() * 1000 ) 
+	{
+		trap_SendServerCommand( ent - g_entities, va( "print \"Your team's status may only be checked every %i seconds.\"", g_teamStatus.Get() ) );
+		return;
+	}
+
+	level.team[ ent->client->pers.team ].lastTeamStatus = level.time;
+
+	tmp = &g_entities[ 0 ];
+	for ( int i = 0; i < level.num_entities; i++, tmp++ ) 
+	{
+		if ( i < MAX_CLIENTS && tmp->client && tmp->entity ) 
+		{
+			auto health = tmp->entity->Get<HealthComponent>();
```

Comment:
better to use `const auto&` since:

1. you do not modify it
2. no need to do a copy

Reply:
Get<> will return a pointer, so this is fine